internal/server/config/config.go

```diff
@@ -23,6 +23,7 @@ const (
 	DefaultChangelogHorizonOffset           = 0
 	DefaultResolveNodeLimit                 = 25
 	DefaultResolveNodeBreadthLimit          = 100
+	DefaultUsersetBatchSize                 = 1000
```

Comment:
**Question:** Does this need to be a configured value at the server level? Seems like this would be difficult to advise people on how to configure and further, a sensible default could go a long way. 

However, I could see this being configured on a per-datastore basis.

Reply:
Yeah i left it out on purpose from it being an actual server flag because (a) I was lazy to do it here, and (b) I don't know if we want to expose that flag to users yet... at the very least it's going to be very hard to explain what values to use because we don't even know ourselves yet.